config/tests/fixtures/full.env

```diff
@@ -4,8 +4,6 @@ PRIVATE_KEY_ALGORITHM=ed25519
 PRIVATE_KEY_PAYLOAD=8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb
 P2P_ADDRESS=127.0.0.1:5432
 GENESIS_PUBLIC_KEY=ed01208BA62848CF767D72E7F7F4B9D2D7BA07FEE33760F79ABE5597A51520E292A0CB
-GENESIS_PRIVATE_KEY_ALGORITHM=ed25519
-GENESIS_PRIVATE_KEY_PAYLOAD=8f4c15e5d664da3f13778801d23d4e89b76e94c1b94b389544168b6cb894f84f8ba62848cf767d72e7f7f4b9d2d7ba07fee33760f79abe5597a51520e292a0cb
```

Comment:
Here, also, if you remove private key variables, I would logically expect you to add `GENESIS_SIGNATURE` here instead.

Reply:
And, by the way, `full.env`'s purpose is to contain __all__ envs supported by Iroha config.